qubit-sdk/src/main/java/com/qubit/android/sdk/api/InitializationBuilder.java

```diff
@@ -17,12 +17,14 @@ interface SdkConsumer {
   }
 
   private final SdkConsumer sdkConsumer;
+  private final String deviceId;
   private Context appContext;
   private String trackingId;
   private QBLogLevel logLevel;
 
-  InitializationBuilder(SdkConsumer sdkConsumer) {
+  InitializationBuilder(SdkConsumer sdkConsumer, String deviceId) {
```

Comment:
I would move setting custom deviceId to the optional method of InitializationBuilder.

For example: withCustomDeviceId

Reply:
There was a discussion in document: https://docs.google.com/document/d/1ajKBP1bIg6UCs48ItgiLxMu5iJR_Q-3xqUksuh58III/edit?disco=AAAAcjkzi3I
Conclusion is that it would be better to add setting customDeviceId to initialization.